packages/app/cypress/e2e/integration/navigation.spec.ts

```diff
@@ -0,0 +1,20 @@
+import defaultMessages from '../../../../frontend-shared/src/locales/en-US.json'
+import type { Interception } from '../../../../net-stubbing/lib/external-types'
+
```

Comment:
Same here!

Reply:
For some reason I did not expect the aliases to work like that from an e2e spec, but they do 👍